git_machete/tests/functional/test_machete.py

```diff
@@ -1822,6 +1823,7 @@ def test_github_create_pr(self) -> None:
         {'head': {'ref': 'ignore-trailing'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'hotfix/add-trigger'}, 'number': '3', 'html_url': 'www.github.com'}
     ])
 
+    # We need to mock GITHUB_REMOTE_PATTERNS in the tests for `test_checkout_prs` due to `git fetch` executed by `checkout-prs` subcommand.
```

Comment:
This one is okay coz you've explained the underlying mechanism to me, it doesn't seem that any bug is lurking here, unlike with `create-pr` above